src/IROperator.cpp

```diff
@@ -567,7 +567,7 @@ Expr halide_erf(Expr x_full) {
     return result;
 }
 
-Expr raise_to_integer_power(Expr e, int p) {
+Expr raise_to_integer_power(Expr e, int64_t p) {
```

Comment:
out of curiosity, why is this necessary? (ie where are we calling this with int64)